frontend/src/components/common/SimpleTable.tsx

```diff
@@ -153,6 +154,7 @@ export default function SimpleTable(props: SimpleTableProps) {
     columns,
     data,
     filterFunction = null,
+    actionFilterFunction = null,
```

Comment:
I'd rather keep a single filter function. This seems too intimate to this PR's use-case and it's not clear what exactly `actionFilterFunction` does or why it's needed.

Reply:
there is ```filterFunc``` which uses the search capabilities from store using hooks like ```useFilterFunc()```  but then we need another filterFunction which works beside the normal filtering, that's why this new prop

Reply:
I forgot to insist on getting a solution for this. I will try to fix it in another PR.

Reply:
@ashu8912 I have added the fix in #979 . Please take a look.